### ISSUE TITLE
Facepalm

### ISSUE THREAD

**Issue by marconi** [jawbreakyr/dragon-test#2]
Never commit credentials! https://github.com/jawbreakyr/dragon-test/blob/master/epurchase/views.py#L23 consider it compromised. :)

**Comment by name3anad** [comment]
Yes, that is my bad. I didn't plan to commit the change when i made it but
hubert needed that change so i committed without thinking. My bad. Thanks
for the help. Lesson learned.
